frontend/src/utils/mock_club_data.ts

```diff
@@ -0,0 +1,64 @@
+export const CLUB_NAMES = [
```

Comment:
What about creating a Club object and specifying those as one list? Same way you do it for Books (instead of BOOKS_TITLES, BOOKS_AUTHORS, and BOOKS_ISBNS)